src/pages/components/notification/usage.mdx

```diff
@@ -4,79 +4,166 @@ description: Notifications are messages that communicate information to the user
+
+### Placement
+
+#### Inline notifications
+
+Inline notifications appear near their related items. In forms, we recommend placing the inline notification at the bottom of the form, right before the submission buttons. Depending on the context of the page, inline notifications can appear above the content as well.
 
 <Row>
 <Column colLg={8}>
 
-![High vs. low contrast](images/notification-usage-4.png)
+![Example of inline notification placement](images/notification-placement-inline.png)
 
 </Column>
 </Row>
 
-## Placement
-
 #### Toast notifications
 
-Toast notifications slide in and out a page from the top-right corner. Actionable notifications do not appear on mobile screen widths.
+Toast notifications slide in and out from the top right of the screen. They align to the edge of the last column and can stack with `$spacing-03` in-between. New toast notifications should appear at the top of the list, with older notifications being pushed down until they are dismissed.
+Actionable toast notifications do not appear on mobile screen widths.
 
 <Row>
 <Column colLg={8}>
 
-![Toast notification example](images/notification-usage-2.png)
+![Example of toast notification placement](images/notification-placement-toast.png)
 
 </Column>
 </Row>
 
-#### Inline notifications
+## Behavior
+
+### Dismissal
+
+We recommend that toast notifications automatically disappear after five seconds. Inline notifications are persistent until the user dismisses them. All notifications have at least one method of dismissal which is typically a small “x” in the upper right hand corner.
+
+### Overflow
+
+Notification messages should be kept short. For toast notifications, the timestamp is optional and can be removed if a third line of content is needed.
+
+If a toast or inline notification requires a message of longer than three lines, include a short message with a “View more” link that takes the user to a view of the full notification message. This can be either a full page with more details or a modal.
+
+## Content
+
+Notifications provide limited space for content, and therefore the content must be concise. A user should be able to quickly scan the notification, be apprised of the situation, and know what to do next.
+
+### Title
+
+- The title should be short and descriptive, explaining the most important piece of information.
+- When possible, communicate the main message using just the title.
+- Use a period only if the title is a full sentence.
 
-Inline notifications appear near its related item. In [forms](/components/form/code), we recommend placing the inline notification at the bottom of the form, right before the submission buttons. Depending on the context of the page, inline notifications can appear above the content as well.
+### Body content
+
+- The body of the notification should be no more than two lines.
```

Comment:
How are we suggesting folks tackle this in the globalization/overflow use-case?

Reply:
Yeah good point with translation we can't really control the line length. We could make these comments less line specific. Like "Content should be 1-2 short sentences." I do think we should emphasize somehow that it shouldn't have excess content. I could even say one short sentence to emphasize that.